core/src/AzureMcp.Cli/Program.cs

```diff
@@ -83,6 +83,7 @@ private static IAreaSetup[] RegisterAreas()
             new AzureMcp.BicepSchema.BicepSchemaSetup(),
             new AzureMcp.Cosmos.CosmosSetup(),
             new AzureMcp.Monitor.MonitorSetup(),
+            new AzureMcp.MySql.MySqlSetup(),
```

Comment:
Hi @ramnov, 

1. Was MySQL moved to non-native due to trim errors from the `Azure.ResourceManager.MySQL` package (1), or was it because of trim-errors with the external `MySqlConnector` package (2)? If it's reason 1, we should be fine, please confirm.

Also, for consistency, if we're excluding MySQL here, it should also be excluded in `AzureMcp.cli.csproj` under the same `BuildNative == true` condition, this ensures consistency.

Reply:
I tested the PR locally, the trim errors originate from `Azure.ResourceManager.MySql` package we own. I've added it here: `https://github.com/Azure/azure-sdk-for-net/issues/51540`.

Please address the second feedback about consistency. thank you!

Reply:
@anuchandy addressed in this commit https://github.com/Azure/azure-mcp/pull/855/commits/82f7b8af3a69d6b2e44eacc015c729224d043408